client/extensions/woocommerce/state/ui/products/edits-reducer.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { isEqual, uniqueId } from 'lodash';
+import { isEqual, isNumber, uniqueId } from 'lodash';
```

Comment:
`isNumber` doesn't get used, and is causing a linter error.

`'isNumber' is defined but never used`